completions/ip

```diff
@@ -384,12 +384,30 @@ _comp_cmd_ip()
                 add | identify | list-id)
                     # TODO
                     ;;
-                delete | exec | pids | set)
+                delete | pids | set)
                     [[ $prev == "$subcmd" ]] && _comp_cmd_ip__netns "$1"
                     ;;
+                exec)
+                    local is_all=0
+                    for ((i = 1; i <= cword; i++)); do
```

Comment:
I believe `i` needs to be made `local` to avoid a leak. It is made so further down below, but in a hunk of code where it is not being used.